modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -596,7 +603,14 @@ export default class I3SConverter {
       return traversalProps;
     }
     if (sourceTile.id) {
-      console.log(`[convert]: ${sourceTile.id}`); // eslint-disable-line
+      // Print the tile number that is currently being converted.
+      this.tileCountCurrentlyConverting++;
+      let percentString = '';
+      if (this.tileCountTotal) {
+        const percent = (this.tileCountCurrentlyConverting / this.tileCountTotal) * 100;
+        percentString = percent.toFixed(this.numberOfDigitsInPercentage);
+      }
+      console.log(`[convert]:[${percentString}%]: ${sourceTile.id}`); // eslint-disable-line
```

Comment:
I would make more compact style:
```suggestion
      console.log(`[convert ${percentString}%]: ${sourceTile.id}`); // eslint-disable-line
```

Reply:
Done